traveler-swift-core/TravelerKit/Models/QueryItem.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  QueryItem.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-10-04.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// An item in the `CatalogGroup` that contains a query
+public struct QueryItem: CatalogItem, Decodable {
+    /// Secondary title
+    public let subTitle: String?
+    /// Title
+    public let title: String
+    /// URL for a thumbnail
+    public let imageURL: URL?
+    /// A `QueryType`
+    public let type: QueryType
+    /// A `SearchQuery` 
+    public var query: SearchQuery
```

Comment:
Do we need both of these here??

I think we can deduce it to just one attribute